pkg/agent/agent.go

```diff
@@ -114,13 +115,20 @@ func (a *Agent) Run(ctx context.Context) error {
 		)
 
 		for {
-			as, err = a.attest(ctx, sto, cat, metrics, nodeAttestor)
-			if err == nil {
-				break
+			InsecureBootstrap := false
+			BootstrapTrustBundle, err := sto.LoadBundle()
```

Comment:
The variables InsecureBootstrap and BootstrapTrustBundle are declared with an uppercase first letter, which is typically reserved for exported variables. Consider renaming them to insecureBootstrap and bootstrapTrustBundle.